server/src/main/java/org/elasticsearch/Build.java

```diff
@@ -103,7 +113,12 @@ private static Build findLocalBuild() {
             );
         }
 
-        return new Build("default", type, hash, date, isSnapshot, version);
+        final String flavor = "default";
+        String minWireCompat = Version.CURRENT.minimumCompatibilityVersion().toString();
+        String minIndexCompat = Version.CURRENT.minimumIndexCompatibilityVersion().toString();
```

Comment:
`minimumIndexCompatibilityVersion` is hopefully going away soon - can `IndexVersion.MINIMUM_COMPATIBLE` be used instead?

Reply:
Hrm, actually this is a semantic change to the version string returned (changes from semver to single integer). I'm not saying we shouldn't change that, but i think this change should not change anything about the response, and we can discuss separately if that change is ok?

Reply:
sure